CHANGELOG.md

```diff
@@ -4,12 +4,26 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### [v2.0.0-beta.1](https://github.com/PrefectHQ/prefect-design/compare/v2.0.0-beta.0...v2.0.0-beta.1)
+
+- add rounded-default token [`#907`](https://github.com/PrefectHQ/prefect-design/pull/907)
+- Initial v2 beta release [`#904`](https://github.com/PrefectHQ/prefect-design/pull/904)
+
 #### [v2.0.0-beta.0](https://github.com/PrefectHQ/prefect-design/compare/v2.0.0-0...v2.0.0-beta.0)
 
-#### [v2.0.0-0](https://github.com/PrefectHQ/prefect-design/compare/v1.11.8-beta.0...v2.0.0-0)
+> 14 August 2023
+
+#### [v2.0.0-0](https://github.com/PrefectHQ/prefect-design/compare/v1.90.1000-0...v2.0.0-0)
```

Comment:
Doesn't matter for this commit/pr but you'll want to delete this tag locally so it doesn't keep showing up in the changelog.

Reply:
Ah gotchya, deleted, thanks